@types/@octokit/plugin-throttling/index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module '@octokit/plugin-throttling' {
+  import Octokit from '@octokit/rest'
+  function plugin(octokit: Octokit, options: Octokit.Options): void
+  export = plugin
+}
```

Comment:
I wonder why you don't put an `index.d.ts` into the root folder and set the `"types"` key in package.json? Similar to what we do here https://github.com/octokit/rest.js/blob/9b85cd674741ee67d03f510e19d24a9d14014e02/package.json#L81

Reply:
Hi @gr2m, this is based solely on the link that the issue originally referenced: https://github.com/probot/probot/tree/master/src/%40types/%40octokit/plugin-throttling

Assuming that’s the type, I just opened a PR for it. 

Disclaimer, I’m not a typescript expert and muddling my way through it. Please correct as appropriate.

Thanks!

Reply:
@gr2m, I did some more review and realized the `index.d.ts` I proposed didn't actually provide any interfaces or type definitions at all. I've pushed new commits as well as moved the file to the root folder per your suggestion. Thanks

Reply:
> I’m not a typescript expert and muddling my way through it

Same 😄